package.json

```diff
@@ -74,8 +74,8 @@
     "universal-cookie": "4.0.4"
   },
   "peerDependencies": {
-    "@edx/frontend-build": ">= 8.1.0 || ^12.9.0-alpha.1",
-    "@edx/paragon": ">= 10.0.0 < 22.0.0",
+    "@openedx/frontend-build": ">= 8.1.0 || ^12.9.0-alpha.1",
```

Comment:
[@openedx/frontend-build](https://www.npmjs.com/package/@openedx/frontend-build?activeTab=versions) doesn't have [12.9.0-alpha.7](https://www.npmjs.com/package/@edx/frontend-build/v/12.9.0-alpha.7) build. It would crash for repos that use frontend-build alpha

Reply:
I updated the versions based on the lowest released versions in the openedx scope https://github.com/openedx/frontend-platform/compare/858ddad3f0490e3e40e992fb441b1ab57ba0a9cf..199a4b500583ed8693c77d44749d3bb19b4a6492